### PR TITLE
Create client.fa_IR.yml

### PR DESCRIPTION
**Pull request by Qasem-h**

### CODE REVIEW

**Comment by Qasem-h** [comment]
@eviltrout
Please review this PR

File: config/locales/client.fa_IR.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Read more here: https://meta.discourse.org/t/contribute-a-translation-to-discourse/14882
+# Persian language translate By http://discours.ir
```
Comment (eviltrout) [inline config/locales/client.fa_IR.yml]: I don't think this comment is necessary.

**Comment by Qasem-h** [comment]
Ok im chenged it

**Comment by eviltrout** [comment]
Thanks

**Pull request merged**